src/lib/device-config.ts

```diff
@@ -95,7 +95,7 @@ export const generateConfig = async (
 		{
 			appUpdatePollInterval:
 				_.parseInt(req.param('appUpdatePollInterval')) * 60 * 1000,
-			network: req.param('network'),
+			network: req.body.network ?? req.query.network,
```

Comment:
The type error here was flagged up after the change from `Array<>` to `any[]` in typings, so I've converted this away from the deprecated `req.param` to fix it